fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -35,20 +36,54 @@
  * @since 6.0.0
  */
 @Configuration
+@PropertySource (value = "file:${" + FedoraPropsConfig.FCREPO_CONFIG_FILE + "}", ignoreResourceNotFound = true)
 public class FedoraPropsConfig {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FedoraPropsConfig.class);
 
+    public static final String FCREPO_CONFIG_FILE = "fcrepo.config-file";
     public static final String FCREPO_HOME = "fcrepo.home";
-
+    public static final String FCREPO_JMS_HOST = "fcrepo.jms.host";
+    public static final String FCREPO_DYNAMIC_JMS_PORT = "fcrepo.dynamic.jms.port";
+    public static final String FCREPO_DYNAMIC_STOMP_PORT = "fcrepo.dynamic.stomp.port";
+    public static final String FCREPO_ACTIVEMQ_CONFIGURATION = "fcrepo.activemq.configuration";
+    public static final String FCREPO_NAMESPACE_REGISTRY = "fcrepo.namespace.registry";
+    public static final String FCREPO_EXTERNAL_CONTENT_ALLOWED = "fcrepo.external.content.allowed";
     private static final String DATA_DIR = "data";
+    private static final String ACTIVE_MQ_DIR = "ActiveMQ/kahadb";
+    private static final String FCREPO_ACTIVEMQ_DIRECTORY = "fcrepo.activemq.directory";
```

Comment:
These variables are getting confused. Some of them are property names and some of them are default property values. They should be visually separate in the file.

Reply:
I will change that up.